MockDataNaming.md

```diff
@@ -21,6 +21,7 @@ The base name consists of:
 - if there is a query string:
 	- a question mark `?`
 	- the query string (it is expected that the keys and values in a standard query string will be [percent-encoded](http://en.wikipedia.org/wiki/Percent-encoding) )
+    - if the query string would make the filename too long for HFS+ (255 character max) the SHA-256 hash of the query will be used instead
```

Comment:
I realize this is being pedantic, but let's add "as a 64-character lower-case hexadecimal string" and clarity that the query should be %-encoded first:

>     - if the percent-encoded query string would make the filename too long for HFS+ (255 character max) the SHA-256 hash of the percent-encoded query string (as a 64-character lower-case hexadecimal string) will be used instead